src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -362,7 +268,13 @@ Polymer({
     return valid ? '' : this.localize('error-keys-port-bad-input');
   },
 
-  _getShortName: getShortName,
+  _isDo(cloudId) {
+    return cloudId === DO_CLOUD_ID;
```

Comment:
No need for cloud id. Pass the cloud name and icon.

Perhaps we need a cloud_assets library with methods like getDigitaloceanIcon()

Reply:
OK, added a cloud_assets library with `getCloudIcon(DisplayCloudId)`.